### PR TITLE
Update Cloud Shell tutorial

### PR DESCRIPTION
**Pull request by kelsk**
**Change summary**
Update sections of the Cloud Shell interactive tutorial, in accordance with changes made to the Cloud Shell Editor UI.

**View staged tutorial**
Review the staged changes to the tutorial in Cloud Shell: [Open in Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor?show=ide&cloudshell_git_repo=https://github.com/kelsk/bank-of-anthos&cloudshell_git_branch=cloudshell-tutorial&cloudshell_workspace=.&cloudshell_tutorial=extras/cloudshell/tutorial.md)

### CODE REVIEW

File: extras/cloudshell/tutorial.md [hunk 1]
```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
...
+<walkthrough-project-setup></walkthrough-project-setup>
 
-1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+## Kubernetes cluster setup
 
-2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+This section shows you how to set up a Kubernetes cluster using either **minikube** or **GKE**.
 
-3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+1. Navigate to the <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer">**Clusters**</walkthrough-editor-spotlight> explorer in the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes sidebar</walkthrough-editor-spotlight> using the left side Activity bar.
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:24]: I'd reword it so the first spotlight that shows is the first one to be clicked. Currently they're in reverse order. e.g. something along the lines of "in the **cloud code - kubernetes** tab of the activity bar, navigate to the **clusters** explorer"

Or alternatively (I like this one a bit more), this bullet point could just be to "navigate to the **cloud code - kubernetes** tab" whereas in the next bullet point, we'd prefix with "in the **clusters explorer**, click **+** in the title bar... etc etc"

(spotlights in bold)

File: extras/cloudshell/tutorial.md [hunk 2]
```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
...
 
-Once minikube has finished starting, you're ready to move on to the next step.
+2. Click <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer-add-cluster">'+'</walkthrough-editor-spotlight> in the title bar of the Clusters explorer to create a new cluster.
 
+3. Choose either **Minikube** or **Google Kubernetes Engine** and follow the instructions below.
 
---- 
 
-### GKE
+### Minikube
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:31]: I wonder if it wouldn't be worth adding a tiny disclaimer under this and the next header, e.g. "if you're planning on deploying in GKE, skip this section"

File: extras/cloudshell/tutorial.md [hunk 3]
```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
...
 
-In order to create a GKE cluster, you'll need to **[create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)** or use an existing project.
+4. Select the **minikube** profile.
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:33]: On my end "Minikube" is capitalized in Cloud Code

File: extras/cloudshell/tutorial.md [hunk 4]
```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
...
-1. Access the command palette by going to **View > Find Command**.
+5. Select **Start**. Cloud Code will initiate a minikube cluster.
+
+6. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+
+
+### GKE
 
-2. Run the command **"Cloud Code: Create GKE cluster"**.
+4. Click **+ Create a new GKE Cluster**.
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:42]: Before this step, there should be a "select the google kubernetes engine profile" bullet point to be consistent with the minikube section

Comment (bourgeoisor) [reply extras/cloudshell/tutorial.md:42]: Just noticing that there's already a common step that says "pick either minikube or gke", in which case, I'd either remove the common step, _or_ I'd remove the "select the minikube profile" altogether, since if we keep both the step would be repeated.

File: extras/cloudshell/tutorial.md [hunk 5]
```diff
@@ -100,15 +105,16 @@ To stop running the app:
...
 
 You can start, stop, and debug apps from the Debug view.
 
+### Cleanup
 If you've deployed your app to a GKE cluster in your Google Cloud Platform project, you'll want to delete the cluster to avoid incurring charges.
 
 1. Navigate to the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes view</walkthrough-editor-spotlight> in the Activity bar.
 
-2. Under the <walkthrough-editor-spotlight spotlightId="cloud-code-gke-explorer">Google Kubernetes Engine Explorer tab</walkthrough-editor-spotlight>, right-click on your cluster and select **Delete Cluster**.
+2. Under the <walkthrough-editor-spotlight spotlightId="cloud-code-gke-explorer">Clusters Explorer tab</walkthrough-editor-spotlight>, right-click on your cluster and select **Delete Cluster**.
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:117]: In some bits of the tutorial it's "**clusters** explorer tab" and here it's "**clusters explorer tab**" (spotlights in bold). Let's make sure what text is being spotlighted (and capitalization / etc) is consistent throughout the tutorial

File: extras/cloudshell/tutorial.md (hunk 1 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:24]: I like your alternate option, to not just fix the order but to move the "Clusters explorer" to the next bullet point! Updated ✅

File: extras/cloudshell/tutorial.md (hunk 2 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:31]: That's a good idea. I found it difficult to format the two options in a way that makes it clear the user only chooses one or the other.

Updated with: *(skip this section if you're planning on deploying to GKE)*

File: extras/cloudshell/tutorial.md (hunk 3 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:33]: For me, the first Minikube prompt is capitalized (the prompt to pick between Minikube & GKE), but the second minikube prompt is lower case ("select a minikube profile"). Is that different in your version? 

(I also think Cloud Code should update both "minikube"s to be consistent - I'll file that as a bug, and update here if they change the UI)

File: extras/cloudshell/tutorial.md (hunk 4 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:42]: Ah, there is a separate step to select the profile for minikube, but there isn't one for GKE.

Profiles are minikube's way of naming and managing clusters, and selecting a minikube profile is equivalent to selecting a GKE cluster. 

And Cloud Shell only has one minikube profile by default - "minikube"! So it looks like a redundant step in the instructions. I've added a link to the minikube docs about profiles to make it more clear that this is a unique step, do you think this is sufficient?

File: extras/cloudshell/tutorial.md (hunk 5 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:117]: Fixed! :)

I also changed "**Cloud Code - Kubernetes sidebar**" to "**Cloud Code - Kubernetes** sidebar" (I double checked other tutorials to see which one is used so that this tutorial can be consistent)

File: extras/cloudshell/tutorial.md [hunk 6]
```diff
@@ -9,44 +9,50 @@ You'll be able to run Bank of Anthos on:
...
-2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+This section shows you how to set up a Kubernetes cluster using either **minikube** or **GKE**.
 
-3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+1. Open the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes</walkthrough-editor-spotlight> sidebar using the left side Activity bar.
 
-Once minikube has finished starting, you're ready to move on to the next step.
+2. In the <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer">**Clusters**</walkthrough-editor-spotlight> explorer, click <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer-add-cluster">'+'</walkthrough-editor-spotlight> in the title bar to create a new cluster.
 
+3. Choose either **Minikube** or **Google Kubernetes Engine** and follow the instructions below.
```
Comment (bourgeoisor) [inline extras/cloudshell/tutorial.md:28]: Maybe "and following the relevant instructions below" to emphasis that you're not following _all_ of the instructions below

File: extras/cloudshell/tutorial.md (hunk 2 above)
Comment (bourgeoisor) [reply extras/cloudshell/tutorial.md:31]: Should we also add a similar note in the GKE section? (skip this if you're deploying to minikube)

File: extras/cloudshell/tutorial.md (hunk 4 above)
Comment (bourgeoisor) [reply extras/cloudshell/tutorial.md:43]: Okay that makes sense!!

File: extras/cloudshell/tutorial.md (hunk 5 above)
Comment (bourgeoisor) [reply extras/cloudshell/tutorial.md]: Thank you!!

File: extras/cloudshell/tutorial.md (hunk 3 above)
Comment (bourgeoisor) [reply extras/cloudshell/tutorial.md]: Oh yeah, just seeing that now, okay sounds good!!

File: extras/cloudshell/tutorial.md (hunk 2 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:31]: Good idea!! Added ✅

File: extras/cloudshell/tutorial.md (hunk 6 above)
Comment (kelsk) [reply extras/cloudshell/tutorial.md:28]: Added! ✅

**Review by bourgeoisor — APPROVED** [review]
LGTM! Thank you so much for making these changes!!
